wallet-new/docs/spec.tex

```diff
@@ -218,6 +226,36 @@ \subsection{Notation}
 \end{split}
 \end{equation*}
 
+\subsubsection{Properties}
+
+Domain restriction distributions over union:
+%
+\begin{equation} \label{restrictdom_union}
+\begin{split}
+& a \restrictdom (u \cup v) \\
+= & \{ x \mapsto y ~|~ x \mapsto y \in (u \cup v), x \in a \} \\
+= & \{ x \mapsto y ~|~ (x \mapsto y \in u) \vee (x \mapsto y \in v), x \in a \} \\
+= & \{ x \mapsto y ~|~ (x \mapsto y \in u, x \in a) \vee (x \mapsto y \in v, x \in a) \} \\
+= & \{ x \mapsto y ~|~ x \mapsto y \in u, x \in a \} \cup \{ x \mapsto y ~|~ x \mapsto y \in v, x \in a \} \\
+= & (a \restrictdom u) \cup (a \restrictdom v)
+\end{split}
+\end{equation}
+%
+and similarly for domain exclusion
+%
+\begin{equation} \label{subtractdom_union}
+a \subtractdom (u \cup v) = (a \subtractdom u) \cup (a \subtractdom v)
+\end{equation}
+
+Both domain restriction and exclusion affect the domain only:
+%
+\begin{align}
+(a \cap \dom u) \restrictdom u & = a \restrictdom u \label{restrictdom_dom} \\
+(a \cap \dom u) \subtractdom u & = a \subtractdom u \label{subtractdom_dom}
+\end{align}
+
+\todo[inline]{Use consistent variable naming.}
+
```

Comment:
These are (improved descriptions of) the lemmas from the other section right? So shouldn't we also delete them from that section and update the label references?

Reply:
I'm not sure I follow. Are you saying you already stated these lemmas?

Reply:
My mistake. They're different lemmas.